interface/orders/list_reports.php

```diff
@@ -166,404 +166,424 @@ function openPatient(pid) {
 </script>
```

Comment:
Where do I trigger this area...

Reply:
In Procedures > Electronic Reports

I guess it needs to import the HL7 file. 

cc @bradymiller, @stephenwaite